packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -35,6 +35,10 @@ export namespace WorkspaceCommands {
         id: 'workspace:open',
         label: 'Open...'
     };
+    export const OPEN_RECENT: Command = {
```

Comment:
[minor]: `OPEN_RECENT_WORKSPACE`? What do you think?

Reply:
agree